### ISSUE TITLE
Empty value breaks parsing ethtool output

### ISSUE THREAD

**Issue by hafkensite** [Showmax/prometheus-ethtool-exporter#8]
On some of our systems the ethtool -m output is like:
```
$ sudo ethtool -m enp33s0f0np0 
	Identifier                                : 0x03 (SFP)
	Extended identifier                       : 0x04 (GBIC/SFP defined by 2-wire interface ID)
	Connector                                 : 0x07 (LC)
...
	Vendor name                               : FiberStore
	Vendor OUI                                : 00:00:00
	Vendor PN                                 : SFP-10GSR-85
	Vendor rev                                : 
	Option values                             : 0x00 0x1a
...
```
As can be seen, the `Vendor rev` value is absent. This breaks the parsing in `update_xcvr_info`, it is looking for `: ` using `split()` but that fails with a `ValueError`. I think it could be fixed by splitting only on `":"`, but unsure about possible side effects from that.

**Comment by zloo** [comment]
Thank you for letting us know. I'll discuss this with @westwardharbor0 and we'll fix that in the coming days.

**Comment by zloo** [comment]
@hafkensite Issue should be fixed in the [latest release (v0.5.1)](https://github.com/Showmax/prometheus-ethtool-exporter/releases/tag/0.5.1). Please test it and let us know if you find any other issues.
